From 3fd6b79895ea20ec0681799be43d94662ad31192 Mon Sep 17 00:00:00 2001
From: Mikhail Kurinnoi <viewizard@viewizard.com>
Date: Wed, 21 Nov 2018 10:27:15 +0300
Subject: [PATCH] Fix compilation warnings.

Looks like some gl.h don't care about __glext_h_, forced to use
GL_GLEXT_LEGACY for Linux too.
---
 src/core/graphics/opengl.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git src/core/graphics/opengl.h src/core/graphics/opengl.h
index 25a524ba..c01c026e 100755
--- src/core/graphics/opengl.h
+++ src/core/graphics/opengl.h
@@ -37,7 +37,8 @@
 	#include <OpenGL/gl.h>
 	#include <OpenGL/glu.h>
 #else
-	#define __glext_h_ // don't let gl.h include glext.h
+	#define GL_GLEXT_LEGACY
+	#define __glext_h_
 	#include <GL/gl.h>
 	#include <GL/glu.h>
 	#undef __glext_h_
